kmm/kmm-post-data/src/commonMain/kotlin/com/gchristov/newsfeed/kmmpostdata/PostRepository.kt

```diff
@@ -6,15 +6,19 @@ interface PostRepository {
     suspend fun post(
         postId: String,
         postMetadataFields: String = "main,body,headline,thumbnail"
-    ): DecoratedPost
+    ): Post
 
-    suspend fun redecoratePost(post: DecoratedPost): DecoratedPost
-
-    suspend fun cachedPost(postId: String): DecoratedPost?
+    //TODO: It should return the decorated one?
+    // but SqlDelight has the raw post only and decoration
+    // has been moved to usecase
+    suspend fun cachedPost(postId: String): Post?
```

Comment:
@gchristov it feels like the cache should store `DecoratedPost`s, but at the moment this `SqlDelight` model is for Posts only. Worth another task to cache the `DecoratedPost`?